spec/validators/claim/advocate_supplementary_claim_validator_spec.rb

```diff
@@ -7,14 +7,7 @@
 
   let(:claim) { create(:advocate_supplementary_claim) }
 
-  before(:all) do
-    seed_fee_schemes
-    seed_fee_types
-  end
-
-  after(:all) do
-    clean_database
-  end
+  before { seed_fee_schemes }
```

Comment:
i'd would still be tempted to use the `before(:all)` hook - depending on how many tests are running. a before each hook would need to seed for every example.

Reply:
👍 Have amended to use the before(:all) hook in a shared_context 'seeded fee schemes', as suggested above.